nuget/nupkg.build

```diff
@@ -1,97 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
Are we generating the package by using dotnet pack now instead?

Reply:
We do not generate packages yet.